tgui/packages/tgui/interfaces/ChemDispenser.js

```diff
@@ -131,20 +132,36 @@ export const ChemDispenser = (_props, context) => {
         <Section
           title="Status"
           buttons={
-            recording && (
-              <Box inline mx={1} color="red">
-                <Icon name="circle" mr={1} />
-                Recording
-              </Box>
-            )
+            <>
+              {recording && (
+                <Box inline mx={1} color="red">
+                  <Icon name="circle" mr={1} />
+                  Recording
+                </Box>
+              )}
+              <Button
+                icon="book"
```

Comment:
If someone could tell me how to put a conditional here so it works on the parent `machinery/chem_dispenser` but not the child `machinery/chem_dispenser/drinks`, I would be much appreciative. 

It would be much better than